### PR TITLE
Remove usage of ert/util in job_queue_node

### PR DESCRIPTION
**Pull request by eivindjahren**
## Pre review checklist

- [x] Read through the code changes carefully after finishing work
- [x] Make sure tests pass locally (after every commit!)
- [x] Prepare changes in small commits for more convenient review (optional)
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [x] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: src/clib/lib/job_queue/job_node.cpp [hunk 1]
```diff
@@ -177,7 +135,7 @@ void job_queue_node_set_status(job_queue_node_type *node,
         return;
 
     logger->debug("Set {}({}) to {}", node->job_name, node->queue_index,
-                  job_status_names.at(new_status).c_str());
+                  job_status_names.at(new_status));
```
Comment (andreas-el) [inline src/clib/lib/job_queue/job_node.cpp:138]: debug takes std::string and not char * ?

File: src/clib/lib/job_queue/job_node.cpp [hunk 2]
```diff
@@ -209,8 +167,7 @@ ERT_CLIB_SUBMODULE("queue", m) {
         std::optional<std::string> error_msg = std::nullopt;
 
         if (current_status & JOB_QUEUE_RUNNING && !node->confirmed_running) {
-            node->confirmed_running =
-                !node->status_file || fs::exists(node->status_file);
+            node->confirmed_running = !fs::exists(node->status_file);
```
Comment (andreas-el) [inline src/clib/lib/job_queue/job_node.cpp:170]: Is this equivalent with the replaced code?
It used to be true if string was empty OR the file actually existed?

**Review by andreas-el — CHANGES_REQUESTED** [review]
Had some comments we should look into

File: src/clib/lib/job_queue/job_node.cpp (hunk 1 above)
Comment (eivindjahren) [reply src/clib/lib/job_queue/job_node.cpp:138]: I think it works with either through pybind.

File: src/clib/lib/job_queue/job_node.cpp (hunk 2 above)
Comment (eivindjahren) [reply src/clib/lib/job_queue/job_node.cpp:170]: It was for the possibility of setting status_file to nullptr, which we don't do.

File: src/clib/lib/job_queue/job_node.cpp (hunk 1 above)
Comment (andreas-el) [reply src/clib/lib/job_queue/job_node.cpp:138]: Probably.

File: src/clib/lib/job_queue/job_node.cpp (hunk 2 above)
Comment (eivindjahren) [reply src/clib/lib/job_queue/job_node.cpp:170]: Oh shit, I guess we need to fix that. We don't really use that method any more, but we might want to investigate what is going on.

**Review by andreas-el — APPROVED** [review]
Great work! 🚀

**Pull request merged**